Lib/test/test_named_expressions.py

```diff
@@ -470,5 +471,52 @@ def test_named_expression_variable_reuse_in_comprehensions(self):
                 self.assertEqual(ns["x"], 2)
                 self.assertEqual(ns["result"], [0, 1, 2])
 
+    def test_named_expression_global_scope(self):
+        sentinel = object()
+        global GLOBAL_VAR
+        def f():
+            global GLOBAL_VAR
+            GLOBAL_VAR = None
```

Comment:
What happens without this initialization? The example in the [bpo issue](https://bugs.python.org/issue38469) doesn't have this. Do we need another set of tests where there's no other local assignment to the same (global or nonlocal) variable?

Reply:
This was to avoid weird cases in the future when some other test modifies the global to some other thing but is really not needed.